README.md

```diff
@@ -113,244 +112,128 @@ bash test/test_cleanup.sh
-#   and one that starts one nucleotide after the annotated mature miRNA; all
-#   isomiRs will stop one nucleotide after the end of the annotated mature
-#   miRNA; note that because `0` is not included in the `bp_3p` array, the
-#   annotated mature miRNAs will not be included in this example
-# - Use `bp_5p: [0]` and `bp_3p: [0]` to only include the mature annotated
-#   miRNAs and no isomiRs
-
-bp_5p: [-2, -1, 0, +1, +2]
-bp_3p: [-2, -1, 0, +1, +2]
-
-#### PROCESSING PARAMETERS #### 
-
-# quality filter
-q_value: 10  # Q (Phred) score; minimum quality score to keep
-p_value: 50  # minimum % of bases that must have Q quality
+```bash
+snakemake \
+    --snakefile="../workflow/Snakefile" \
+    --report="snakemake_report.html"
+```
 
-# adapter removal
-error_rate: 0.1     # fraction of allowed errors
-minimum_length: 15  # discard processed reads shorter than the indicated length
-overlap: 3          # minimum overlap length of adapter and read to trim the bases
-max_n: 0            # discard reads containing more than the indicated number of N bases
+> **NOTE:** The report creation must be done after running the workflow in
+> order to have the runtime statistics and the results. 
 
-# mapping
-max_length_reads: 30  # maximum length of processed reads to map with oligomap
-nh: 100               # discard reads with more mappings than the indicated number
+## Workflow description
 
-#### QUANTIFICATION PARAMETERS ####
+The _MIRFLOWZ_ workflow starts by processing the user-provided "genome 
+resources" while preparing indexes and other contigent resources that will be
+used in later steps. Afterwards, the user-provided short read smallRNA-seq
+library(ies) will be aligned against the references previously generated. For
+increased fidelity, it uses two separate aligning tools, [Segemehl][segemehl] 
+and our in-house tool [Oligomap][oligomap]. In both cases, reads are aligned
+separately to the genome and the transcriptome. Thereafter, alignments are
+merged in a way that only the best alignment (or alignments) of each read is 
+(are) kept. Finally, the quantification of miRNA expression is done by
+intersecting the alignments with the generated annotation files. Intersections 
+are computed with [`bedtools`][bedtools] for one or multiple of mature, primary
+transcripts and isomiRs. Reads consistent with each miRNA are counted and 
+tabulated.
 
-# Types of miRNAs to quantify
-# Remove miRNA types you are not interested in
-mir_list: ["miRNA", "miRNA_primary_transcript", "isomirs"]
-...
-```
 
-> **Note:** We expect the genome and gene annotations to be formatted according
-> the style used by Ensembl. Other formats are very likely to lead to problems.
-> The miRNA annotation file is expected to originate from
-> miRBase, or follow their exact layout.
+The schema below is a visual representation of the rules building up the 
+workflow and how they are related:
 
+> ![rule-graph][rule-graph]
 
```

Comment:
Would be good to have sections on "Contributing", "License" and "Contact", such as here: https://github.com/elixir-cloud-aai/foca#contributing

Contact email is this: zavolab-biozentrum@unibas.ch

For contributing, just say something generic about welcoming everyone's contributions blabla. No code of conduct, not guidelines (unless you wanna deal with these, up to you).